crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -300,10 +283,96 @@ impl Persistence {
             id: auction_id,
             orders,
             prices,
-            deadline,
             surplus_capturing_jit_order_owners,
         })
     }
+
+    /// Get competition winner.
```

Comment:
Amazing comment 👏 

We could describe here what where we get this data from and how it's assembled but as is it has no value.

Reply:
resolved